community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/staging/DynamicProcessorAssigner.java

```diff
@@ -98,15 +93,15 @@ private int assignProcessorsToPotentialBottleNeck( StageExecution execution, int
         if ( bottleNeck.other() > 1.0f &&
              batchesPassedSinceLastChange( bottleNeckStep, doneBatches ) >= config.movingAverageSize() )
         {
+            // Assign 1/10th of the remaining permits. This will have processors being assigned more
+            // aggressively in the beginning of the run
```

Comment:
that will also mean that in the end when will have less then 10 permits we will always use optimalProcessorIncrement (since it's hard to say what will be result of bottleNeck.other().floatValue() ) i would assume that `optimalProcessorIncrement` in those cases will be always 1, and that will cause a bit of throttling Is that correct?